### ISSUE TITLE
[sklearn#34569] ⚠️ CI failed on Wheel builder (last failure: Aug 09, 2026) ⚠️

### ISSUE THREAD

**Issue by godofecht** [flooooooooooow/flow#315]
**Mirrored from [scikit-learn#34569](https://github.com/scikit-learn/scikit-learn/issues/34569)**

Original title: ⚠️ CI failed on Wheel builder (last failure: Aug 09, 2026) ⚠️
Original labels: Build / CI, OS:Windows
Created: 2026-07-26T05:27:23Z

---

**CI is still failing on [Wheel builder](https://github.com/scikit-learn/scikit-learn/actions/runs/31293828912)** (Aug 09, 2026)

---

## Flow context

This issue is mirrored from scikit-learn to explore whether Flow encounters the same problem or can approach it differently. Add analysis below.

**Comment by godofecht** [comment]
Moved to https://github.com/godofecht/flow-scikit/issues/108 in the flow-scikit repository. Further library-specific triage and implementation will happen there.
